apps/api/src/domain/factories/__tests__/buildAggregateFactoryTestCases/spatial-feature.aggregate-factory.test-cases.ts

```diff
@@ -0,0 +1,76 @@
+import { getValidSpatialFeatureInstanceForTest } from '../../../../domain/domainModelValidators/__tests__/domainModelValidators/buildDomainModelValidatorTestCases/utils/getValidSpatialFeatureInstanceForTest';
+import assertErrorAsExpected from '../../../../lib/__tests__/assertErrorAsExpected';
+import { formatGeometricFeatureType } from '../../../../view-models/presentation/formatGeometricFeatureType';
+import { buildInvalidSpatialFeatureDtoError } from '../../../domainModelValidators/__tests__/domainModelValidators/buildDomainModelValidatorTestCases/spatial-feature.domainModelValidatorTestCase';
+import { Line } from '../../../models/spatial-feature/entities/line.entity';
+import { Point } from '../../../models/spatial-feature/entities/point.entity';
+import { GeometricFeatureType } from '../../../models/spatial-feature/types/GeometricFeatureType';
+import { AggregateType } from '../../../types/AggregateType';
+import { AggregateFactoryValidTestCase, FactoryTestSuiteForAggregate } from './';
+import buildValidCasesForSubtypes from './common/buildValidCasesForSubtypes';
+
+const aggregateType = AggregateType.spatialFeature;
+
+const validPoint = getValidSpatialFeatureInstanceForTest(GeometricFeatureType.point);
+
+const validLine = getValidSpatialFeatureInstanceForTest(GeometricFeatureType.line);
+
+const validCases: AggregateFactoryValidTestCase<typeof aggregateType>[] =
+    buildValidCasesForSubtypes(
+        aggregateType,
+        Object.values(GeometricFeatureType),
+        formatGeometricFeatureType,
+        getValidSpatialFeatureInstanceForTest
+    );
+
+export const buildSpatialFeatureFactoryTestSet = (): FactoryTestSuiteForAggregate<
+    typeof aggregateType
+> => ({
+    aggregateType,
+    validCases,
+    invalidCases: [
+        /**
+         * `Point` invalid test cases
+         */
+        {
+            description: 'one of the coordinates is a string',
+            dto: validPoint
+                .clone<Point>({
+                    geometry: {
+                        type: GeometricFeatureType.point,
+                        coordinates: [12, 'foo' as unknown as number],
+                    },
+                })
+                .toDTO(),
+
+            // TODO [https://www.pivotaltracker.com/story/show/183014405] Check inner errors
```

Comment:
Do this.

Reply:
Or else add a story.

Reply:
There's already a story. Let's do this next time we revisit spatial feature models. It's not important until we have a need to actually add real mapping data.